### PR TITLE
Add some basic properties such as name and desc to package.json

### PR DESCRIPTION
**Pull request by amit-mb**
- Add some basic properties such as name and desc to package.json

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,4 +1,21 @@
 {
+  "name": "interceptor",
+  "version": "0.2.1",
+  "description":
+    "A browser extension that mocks AJAX request at the browser level so you can run front ends without really starting a backend server.",
+  "license": "MIT",
+  "homepage": "https://github.com/code-mancers/interceptor#readme",
+  "repository": {
```
Comment (revathskumar) [inline package.json]: I repository is already there and no need for change

File: package.json [hunk 2]
```diff
@@ -1,4 +1,21 @@
 {
+  "name": "interceptor",
+  "version": "0.2.1",
+  "description":
+    "A browser extension that mocks AJAX request at the browser level so you can run front ends without really starting a backend server.",
```
Comment (revathskumar) [inline package.json]: run **frontend**

File: package.json (hunk 1 above)
Comment (amit-mb) [reply package.json]: i just moved the same thing up. You can notice the deleted lines below  😄

File: package.json (hunk 2 above)
Comment (amit-mb) [reply package.json]: 👍

File: package.json (hunk 1 above)
Comment (revathskumar) [reply package.json]: in url `.git` is missing. 
Ideally this should be the url to `git` repo not its code home page.

**Pull request merged**